bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/UniFiController.java

```diff
@@ -281,6 +281,7 @@ private <T> UniFiControllerRequest<T> newRequest(final Class<T> responseType, fi
             } else {
                 login();
                 result = (T) executeRequest(request);
+                logout();
```

Comment:
Perhaps this was staged by mistake when you wanted to commit the README update?

Reply:
Yes... It's too late apparently :tired_face:

Reply:
Reverted